src/nvim/lua/treesitter.c

```diff
@@ -265,19 +265,25 @@ int tslua_inspect_lang(lua_State *L)
   assert(nsymbols < INT_MAX);
 
   lua_createtable(L, (int)(nsymbols - 1), 1);  // [retval, symbols]
+#define BUFSIZE 256
+  char buf[BUFSIZE];
```

Comment:
can use sizeof(buf)

```suggestion
  char buf[256];
```

Reply:
I think this decl can be moved to line 280 since that is the block it is pushed into Lua.

Reply:
I thought about that, I initially put it outside that block for less stack allocations (but those are cheap ofc...). Do you think readability suffers with it outside that block? I'll move it in if so

Reply:
Our coding rules require moving all variable to the most local scope.